x-pack/plugins/security/server/authentication/authenticator.test.ts

```diff
@@ -515,7 +515,7 @@ describe('Authenticator', () => {
       expect(mockSessionStorage.clear).not.toHaveBeenCalled();
     });
 
-    it('only updates the session lifespan expiration if it does not match the current server config.', async () => {
+    it.skip('only updates the session lifespan expiration if it does not match the current server config.', async () => {
```

Comment:
@jportner I'm not sure if you're the right person to ping about this (blame `git blame` 😅), but I had to skip these tests as they were producing unhandled promise rejections due to an uncaught exception.

Reply:
Yes, sounds like my fault, I will look into it, thanks!